buildah.go

```diff
@@ -340,7 +340,7 @@ type BuilderOptions struct {
 	// needs to be pulled and the image name alone, or the image name and
 	// the registry together, can not be resolved to a reference to a
 	// source image.  No separator is implicitly added.
-	Transport string
```

Comment:
Just wondering why you switched this to lower case?  Also in pull.go.  I understand the scoping, but none of the other variables in these structs are lowercase.  In some of the code we have a number of variables named transport(s) being used and having this upper case helps it to stand out better.

Reply:
I don't want callers to use this field.  We use it to pass the information around withing buildah library, but specifying transport as well as imagename can cause conflicts.  
Bottom line, I only want transport derived in one place and from ImageName.  Before we had it being derived in different places,and not the same way.